src/main/java/com/imgix/URLHelper.java

```diff
@@ -276,8 +276,6 @@ public static String sanitizePath(String path) {
       return "/" + URLHelper.encodeURIComponent(path);
     } else if (pathIsProxy.equals(true) && proxyIsEncoded.equals(true)) {
       return "/" + path;
-    } else if (isASCIIEncoded(path)) {
-      return "/" + path;
```

Comment:
This conditional wasn't needed to support encoded paths. Only the conditional above it.